tensorflow/tools/ci_build/Dockerfile.android

```diff
@@ -29,7 +29,7 @@ RUN mkdir -p ${ANDROID_DEV_HOME}
 ENV ANDROID_SDK_FILENAME tools_r25.2.5-linux.zip
 ENV ANDROID_SDK_URL https://dl.google.com/android/repository/${ANDROID_SDK_FILENAME}
 ENV ANDROID_API_LEVEL 23
-ENV ANDROID_NDK_API_LEVEL 21
+ENV ANDROID_NDK_API_LEVEL 25
```

Comment:
NDK version and NDK version is different. You don't need to update this.

Reply:
Hi @terryheo 

I have reverted the changes back. Thanks.